R/AddCardModule.R

```diff
@@ -32,6 +32,21 @@ add_card_button_ui <- function(id) {
         )
       )
     ),
+    shiny::singleton(
+      shiny::tags$head(
+        shiny::tags$script(
+          shiny::HTML(
+            "
+            $(document).ready(function() {
+              $('body').on('shown.bs.modal', (e) => {
+                $(e.target).find('textarea').focus();
+              });
+            })
+            "
+          )
+        )
+      )
+    ),
```

Comment:
I've updated this to be more specific. Instead of body, it uses `'#shiny-modal` which is the id Shiny uses for modals (bootstrap only allows 1 modal at a time). And instead of `event.target`, I've switched to an id, matching the text area input (`ns("comment")`)